api/control/v1alpha1/types.go

```diff
@@ -443,6 +465,27 @@ const (
 // Type checking errors are posted to the status.Errors of the Binding and BindingPolicy.
 type Expression string
 
+// Augmentation defines what is implicitly added to every workload object from a WEC,
+// for purposes of status collection.
+type Augmentation struct {
+	Inventory   InventoryObjectReference `json:"inventory"`
+	Propagation PropagationMeta          `json:"propagation"`
+}
```

Comment:
This is not a good explanation of the design actually being pursued. The design being pursued does not add fields to "the" object structure. Instead, the top-level names available to expressions are "obj", "returned", "inventory", and "propagation".